### ISSUE TITLE
FA18 JDAM does not detonate on oil platform

### ISSUE THREAD

**Issue by devintrowbridge** [dcs-liberation/dcs_liberation#3422]
### Affected versions

11.1.1

### Build information

DCS Liberation 11.0.0

### Description

tldr tried to bomb some oil platforms in the straight of hormuz using JDAMs, but no explosion and the oil platform remains undamaged.

I generated a mission where I was flying a strike against 4 oil platforms in the middle of the persian gulf. I was equipped with 2 GBU-31(V)2/B 2000lbs JDAMs. We set efuze to INST, put the JDAMs into TOO, and used waypoints as coordinates. Then I verified the coordinates by slaving the ATFLIR to the waypoint and confirmed it was pointing dead center on the oil platforms. I flew at 20k ft, dropped the bombs in the IZLAR, verified via HSI page and HUD. No explosion though.

This is the first turn in a campaign so no `state.json` available.

Troubleshooting steps:

1. To see if the issue was with the munition, I tried doing the JDAM training mission. No issue, got an explosion. 
2. To see if the issue was with the map or the oil platform asset, I created a separate mission in the mission editor. No issue, got an explosion.
3. To see if the issue was with the oil platform, we launched other munitions at it (aim9), and were able to successfully damage it.
5. To see if the issue was the JDAM overshooting or otherwise not contacting the oil platform, I verified via bomb chase cam that it goes through the oil platform model.

### Save game and other files (save game required, bugs without saves will be closed)

[liberation.log](https://github.com/user-attachments/files/16355500/liberation.log)
[liberation_nextturn.zip](https://github.com/user-attachments/files/16355505/liberation_nextturn.zip) << miz file
[dcs.log](https://github.com/user-attachments/files/16355507/dcs.log)
[autosave.liberation.zip](https://github.com/user-attachments/files/16355512/autosave.liberation.zip)
